Dockerfile

```diff
@@ -1,23 +1,26 @@
-FROM node:10-jessie
+FROM --platform=${TARGETPLATFORM:-linux/amd64} node:15.2.1-alpine3.12

```

Comment:
I would recommend to use `node:alpine` here instead. As from https://github.com/Jackett/Jackett/issues/9029#issuecomment-728315266 it looks like Alpine 3.12 does not yet include chromium on ARM platforms.

Reply:
I don't have a problem in using Alpine 3.11 but it's important to use a fixed base image. If they change node or chomium version in the latest image our Dockerfile could stop working.
I'm going to do a test in a real ARM machine to chose the right one.

Reply:
Tested in ARM64: build + run + unprotected url + protected url. All is working in `15.2.1-alpine3.12` !!!!